core/src/save/region/mod.rs

```diff
@@ -344,6 +348,14 @@ fn read_section_into_chunk(section: &LevelSection, chunk: &mut Chunk) -> Result<
 }
 
 fn chunk_to_chunk_root(chunk: &Chunk, entities: Vec<EntityData>) -> ChunkRoot {
+    let heightmaps: Vec<i64> = chunk.heightmaps().iter().map(|map| {
+        (map.motion_blocking() << (9 * 0)
```

Comment:
Could 9 be stored in a constant here?

Reply:
Definitely, however, I'm not sure if I should move the serialization and deserialization to somewhere else.